src/navigation/navigationhandler_lua.inl

```diff
@@ -343,4 +343,20 @@ int addGlobalRoll(lua_State* L) {
     return 0;
 }
 
+int triggerIdleBehavior(lua_State* L) {
+    ghoul::lua::checkArgumentsAndThrow(L, {0, 1}, "lua::triggerIdleBehavior");
+    std::optional<std::string> choice = ghoul::lua::value<std::optional<std::string>>(L);
+
+    try {
+        global::navigationHandler->orbitalNavigator().triggerIdleBehavior(choice);
+    }
+    catch (ghoul::MissingCaseException&) {
```

Comment:
The idea for the `MissingCaseException` is to be a fallback for the cases where the developer forgot to add a case to a switch statement, not as an exception that might occur in regular usage.  I'd suggest to replace it with a ghoul::RuntimeError instead